javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,90 @@
+const inputEl = document.getElementById('inputEl'); 
+const convertFrom = document.getElementById('convertFrom'); 
+const convertTo = document.getElementById('convertTo'); 
+const btn = document.getElementById('btn');
+const displayCurrency = document.getElementById('displayCurrency');
```

Comment:
Some of the variables should be declared as local variables since they are only used in specific functions